packages/front-end/components/Experiment/NewExperimentForm.tsx

```diff
@@ -168,8 +168,16 @@ const NewExperimentForm: FC<NewExperimentFormProps> = ({
   const [conditionKey, forceConditionRender] = useIncrementer();
 
   const attributeSchema = useAttributeSchema();
+
+  const filteredAttributeSchema = attributeSchema.filter((attribute) => {
+    return (
+      !project ||
+      !attribute.projects?.length ||
+      attribute.projects.includes(project)
+    );
+  });
```

Comment:
Should this just pass project into `useAttributeSchema` instead of recreating the filtering logic here?